client/my-sites/plans/index.js

```diff
@@ -10,10 +10,9 @@ import { features, plans, redirectToCheckout, redirectToPlans } from './controll
 import { currentPlan } from './current-plan/controller';
 import { makeLayout, render as clientRender } from 'controller';
 import { navigation, siteSelection, sites } from 'my-sites/controller';
-import { trackNavigationStart } from 'lib/performance-tracking';
 
 const trackedPage = ( url, ...rest ) => {
-	page( url, trackNavigationStart( 'plans' ), ...rest, makeLayout, clientRender );
+	page( url, ...rest, makeLayout, clientRender );
```

Comment:
@scinos After adding and removing LUX, this code ended up in a rather confusing state. There is a function called `trackedPage`, which doesn't do any tracking at all. One needs to do a bit of Git archeology to figure out what's the point.